gen_ci.py

```diff
@@ -106,7 +106,8 @@ def format_apple_clang_build(version):
         xcode_images = {'7.3': 'xcode7.3',
                         '8.1': 'xcode8.3',
                         '9.0': 'xcode9',
-                        '9.1': 'xcode9.4'}
+                        '9.1': 'xcode9.4'
+                        '10.0': 'xcode10'}
```

Comment:
do you know when conan community will update their recipes?

Reply:
Good question. No idea.

Reply:
I'll ask guys, hopefully it will not take long